pkgs/os-specific/linux/kernel/generic.nix

```
@@ -12,8 +12,9 @@
 , rustc
 , rustPlatform
 , rust-bindgen
-
-, # The kernel source tarball.
+, nixosTests
+}:
+{ # The kernel source tarball.
```

Nit: a whitespace makes it clearer.

```suggestion
}:

{ # The kernel source tarball.
```

---

Sure.